docs/reference/glossary.md

```diff
@@ -93,6 +93,27 @@ develop, ship, and run applications
 - The docker daemon process running on the host which manages images and containers
 
 
+## Docker for Mac
+
+[Docker for Mac](https://docs.docker.com/docker-for-mac/) is an easy-to-install,
+lightweight Docker development environment designed specifically for the Mac. A
```

Comment:
Re: `s/the Mac/macOS` I'd like to leave as is for a few of reasons: 1) simpler, more direct 2) not entirely true as up until very recently d4mac was designed for OS X releases, 3) along w/simplicity, just saying "Mac" inherently communicates that the app is designed to work on the hardware (albeit through the OS, but I think "Mac" subliminally gets to that)